src/locales/cy/translation.json

```diff
@@ -1217,6 +1217,51 @@
         }
       }
     },
+    "pagePreDwpKbvTransition": {
+      "title": "Answer security questions",
```

Comment:
Not sure where these translations came from, but this same title is translated as `Ateb cwestiynau diogelwch` in `pagePreExperianKbvTransition`

Reply:
The translation document I was following didnt include the title for some reason, but as you pointed out its the same as Experian so will add that